meiga/result.py

```diff
@@ -1,21 +1,12 @@
-from typing import (
-    Any,
-    Callable,
-    Generic,
-    NoReturn,
-    Optional,
-    Type,
-    TypeVar,
-    Union,
-    cast,
-)
+from typing import Any, Callable, Generic, Optional, Type, TypeVar, Union, cast
```

Comment:
The indentation and newlines seems to be going back and forth here; which is the correct one? :slightly_smiling_face: :

Reply:
We follow the conventions of `black`, `flake8` and `isort` packages (https://github.com/alice-biometrics/meiga/blob/main/.pre-commit-config.yaml). This is automatic!